docs/self-managed/backup-restore/zeebe-backup-and-restore.md

```diff
@@ -10,22 +10,25 @@ A backup of a Zeebe cluster is comprised of a consistent snapshot of all partiti
 Zeebe provides a REST API to create backups, query, and manage existing backups.
 The backup management API is a custom endpoint `backups`, available via [Spring Boot Actuator](https://docs.spring.io/spring-boot/docs/2.7.x/reference/htmlsingle/#actuator.endpoints). This is accessible via the management port of the gateway. The API documentation is also available as [OpenApi specification](https://github.com/camunda/zeebe/blob/main/dist/src/main/resources/api/backup-management-api.yaml).
 
-The backups are stored to an external data storage. [S3](https://aws.amazon.com/s3/) or any S3-compatible storage is supported as the backup storage.
+## Configuration
 
-## Prerequisites
+To use the backup feature in Zeebe, you must choose which external storage system you will use.
+Make sure to set the same configuration on all brokers in your cluster.
 
-To use the backup feature in Zeebe, the following configurations must be provided:
+Zeebe supports [S3](#s3-backup-store) and [Google Cloud Storage (GCS)](#gcs-backup-store) for external storage.
 
-- Ensure the configuration `MANAGEMENT_ENDPOINTS_WEB_EXPOSURE_INCLUDE` includes the endpoint "backups". By default, this is set to "\*" which included backups.
-- Set `MANAGEMENT_ENDPOINTS_BACKUPS_ENABLED` to `true`.
-- For additional configurations such as security, refer to the [Spring Boot documentation](https://docs.spring.io/spring-boot/docs/2.7.x/reference/htmlsingle/#actuator.endpoints).
-- An external S3-compatible storage must be configured as the backup store.
+:::caution
+Backups created with one store are not available or restorable from another store.
 
-### Configure S3 backup store
+This is especially relevant if you were using GCS through the S3 compatibility mode and want to switch to the new native support for GCS now.
+Even when the underlying storage bucket is the same, backups from one are not compatible with the other.
+:::
 
-Configure backup store in the configuration file as follows:
+### S3 backup store
 
-```
+To store your backups in any S3 compatible storage system such as [AWS S3] or [MinIO], set the backup store to `S3` and tell Zeebe how to connect to your bucket:
```

Comment:
Did you mean to link to AWS/MinIO directly here?

Reply:
Yes, the links are here: https://github.com/camunda/camunda-platform-docs/pull/1882/files#diff-f1d372d47757830aae7a31653eae6b283c78a1a0d19719022886f6d7a1f47cf2R56-R57